src/test/java/com/epam/tc/hw3/exercise2/SiteSelectorsTests.java

```diff
@@ -0,0 +1,174 @@
+
+        checkboxName = "Wind";
+        checkboxLogValues = differentElements.doubleClickOnCheckBoxAndCollectStatusAndLogs(checkboxName);
+        assertThatCheckboxStatusCorrespondingToSelected(checkboxName, checkboxLogValues, softAssertions);
+
+        checkboxName = "Fire";
+        checkboxLogValues = differentElements.doubleClickOnCheckBoxAndCollectStatusAndLogs(checkboxName);
+        assertThatCheckboxStatusCorrespondingToSelected(checkboxName, checkboxLogValues, softAssertions);
+
+        // assert radioButtons
+        String radioButtonName = "Gold";
+        Boolean radioButtonIsSelected = differentElements.radioButtonClick(radioButtonName);
+        assertThatRadioButtonSelectedAndLogIsCorrect(radioButtonName, radioButtonIsSelected,
+            differentElements.getFirstLogMessageAsText(), softAssertions);
+
+        radioButtonName = "Silver";
+        radioButtonIsSelected = differentElements.radioButtonClick(radioButtonName);
+        assertThatRadioButtonSelectedAndLogIsCorrect(radioButtonName, radioButtonIsSelected,
+            differentElements.getFirstLogMessageAsText(), softAssertions);
+
+        radioButtonName = "Bronze";
+        radioButtonIsSelected = differentElements.radioButtonClick(radioButtonName);
+        assertThatRadioButtonSelectedAndLogIsCorrect(radioButtonName, radioButtonIsSelected,
+            differentElements.getFirstLogMessageAsText(), softAssertions);
+
+        radioButtonName = "Selen";
+        radioButtonIsSelected = differentElements.radioButtonClick(radioButtonName);
+        assertThatRadioButtonSelectedAndLogIsCorrect(radioButtonName, radioButtonIsSelected,
+            differentElements.getFirstLogMessageAsText(), softAssertions);
+
+        // assert dropdown
+        String dropDownElementName = "Red";
+        String selectedDropdown = differentElements.dropDownColorsSelect(dropDownElementName);
+        assertThatDropdownSelectedAndLogIsCorrect(dropDownElementName,
+            selectedDropdown, differentElements.getFirstLogMessageAsText(), softAssertions);
+
+        dropDownElementName = "Green";
+        selectedDropdown = differentElements.dropDownColorsSelect(dropDownElementName);
+        assertThatDropdownSelectedAndLogIsCorrect(dropDownElementName,
+            selectedDropdown, differentElements.getFirstLogMessageAsText(), softAssertions);
+
+        dropDownElementName = "Blue";
+        selectedDropdown = differentElements.dropDownColorsSelect(dropDownElementName);
+        assertThatDropdownSelectedAndLogIsCorrect(dropDownElementName,
+            selectedDropdown, differentElements.getFirstLogMessageAsText(), softAssertions);
+
+        dropDownElementName = "Yellow";
+        selectedDropdown = differentElements.dropDownColorsSelect(dropDownElementName);
+        assertThatDropdownSelectedAndLogIsCorrect(dropDownElementName,
+            selectedDropdown, differentElements.getFirstLogMessageAsText(), softAssertions);
```

Comment:
Please read carefully homework task regarding which elements should be checked in a log component

Reply:
Fixed